app/(content)/home-hero.tsx

```diff
@@ -29,7 +30,19 @@ export default function HomeHero() {
             <br />
             zu Hamburg
           </div>
-        </>
+          <ResponsivePositionedBlob
+            id={4}
+            className="fill-green-neon"
+            position={{ top: "47%", left: "94.6%" }}
+            heightPercentage={13.2}
+          />
```

Comment:
This is the same as using arbitrary values in `Blob` `className`, is it not?
```suggestion
          <Blob
            id={4}
            className="fill-green-neon top-[47%] left-[94.6%] h-[13.2%]"
          />
```